app/src/views/AccountDetails/GenerateMontandonTokenModal/index.tsx

```diff
@@ -75,9 +76,10 @@ function GenerateMontandonTokenModal(props: Props) {
                 {
                     termsLink: (
                         <Link
-                            external
-                            href="https://docs.google.com/spreadsheets/d/1kQZQmWk8W4tyE3fhi54Oc2Q-TiLJVUj0/"
-                            withLinkIcon
+                            to="termsAndConditions"
+                            actions={
+                                <ExternalLinkLineIcon />
```

Comment:
This is an internal link. So using `ExternalLinkLineIcon` could be confusing. Let's follow the pattern used in IFRC GO.